app/notification.go

```diff
@@ -286,6 +329,28 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 		}
 	}
 
+	if *a.Config().EmailSettings.SendEmailNotifications {
+		for _, id := range notifyCRT.Email {
+			if profileMap[id] == nil {
+				continue
+			}
+
+			//If email verification is required and user email is not verified don't send email.
+			if *a.Config().EmailSettings.RequireEmailVerification && !profileMap[id].EmailVerified {
+				mlog.Debug("Skipped sending notification email, address not verified.", mlog.String("user_email", profileMap[id].Email), mlog.String("user_id", id))
+				continue
+			}
+
+			senderProfileImage, _, err := a.GetProfileImage(sender)
+			if err != nil {
+				a.Log().Warn("Unable to get the sender user profile image.", mlog.String("user_id", sender.Id), mlog.Err(err))
+			}
+			if err := a.sendNotificationEmail(notification, profileMap[id], team, senderProfileImage); err != nil {
+				mlog.Warn("Unable to send notification email.", mlog.Err(err))
+			}
+		}
+	}
```

Comment:
This whole block is very similar to the previous block. What do you think about combining them a bit?

The primary difference seems to be that for the followers you don't need to check if the user allows emails since it's already been done. I'm thinking something along the lines of creating a new array from `mentionedUsersList` and `notifyCRTEmail`, looping through them all and then just changing the check to be:

```go
if notifyCRT.Email.Contains(id) || a.userAllowsEmail(profileMap[id], channelMemberNotifyPropsMap[id], post) {
```